into/backends/tests/test_sas.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import, division, print_function
+
+import pytest
+sas7bdat = pytest.importorskip('sas7bdat')
+pytest.importorskip('into.backends.sas')
+import os
+import pandas as pd
+import numpy as np
+import datashape
+from collections import Iterator
+from sas7bdat import SAS7BDAT
+
+from into.backends.sas import discover, sas_to_DataFrame, sas_to_iterator
+from into.utils import tmpfile
+from into import append, convert, resource, dshape
+from numpy import dtype
+
+
+cur_path = os.path.abspath(os.path.dirname(__file__))
+test_path = os.path.join(cur_path, 'airline.sas7bdat')
+
+
+@pytest.yield_fixture
+def sasfile():
+    with SAS7BDAT(test_path) as f:
+        yield f
```

Comment:
I'm somewhat against using pytest idioms when we don't have to as it forces users to choose a testing tool.  In this case it seems like we might be able to get away with leaving this file open?  Happy to handle these changes myself.